docs/_layouts/tutorial_frame.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
This is off-topic from the intent of this PR, but I should've mentioned it in https://github.com/Leaflet/Leaflet/issues/7899.